src/common/kits/PilotManager.ts

```
@@ -0,0 +1,563 @@
+  DELAY_STEP = 2000;
+  DELAY_ACTION = 1000;
+  REPEAT_MAX = 5;
+
+  caseUrlMap: Record<string, string> = {
+    "demo": "https://www.gov.uk/skilled-worker-visa/apply-from-outside-the-uk",
+  };
+
+  pilotMap: Map<string, IPilotType> = new Map();
+
+  static getInstance(): PilotManager {
+    if (!this.instance) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)